Task20Locators/TutBy/Tests/TutByTests.cs

```diff
@@ -19,35 +21,23 @@ public void OneTimeSetup()
             LandingPage.OpenLandingPage();
         }
 
-        // Data-driven test using flexible excel reader
         [Test]
         public void LoginWithValidCredentialsTest()
         {
             LandingPage.OpenLoginForm();
             LandingPage.SubmitLoginForm(Dataset.FirstUser);
 
-            Assert.True(LandingPage.WaitFindElement(LandingPage.UsernameLabel).Text == GetFromExcel(Dataset.FirstUser, Field.Username));
```

Comment:
It will be better to determine Username IWebElement on the page level. For instance, if you add line
"public IWebElement UI_UsernameLabel = WaitFindElement(UsernameLabel)" then in test you will just call
"LandingPage.UI_UsernameLabel.Text". So you don't need static page classes. Forget about static!:) Use instance creation of needed page classes.

Reply:
Updated